custom/icds/views/custom_sms_report.py

```diff
@@ -18,26 +19,43 @@ class SMSUsageReport(BaseMessagingSectionView):
 
     @use_daterangepicker
     def dispatch(self, *args, **kwargs):
+        messages.info(self.request, _('Please note that this report takes a few hours to process.'))
         return super().dispatch(*args, **kwargs)
 
     @property
     def page_context(self):
+        report_tracker = CustomSMSReportTracker(self.request.domain)
+        reports_in_progress = report_tracker.active_reports
+        report_count = len(reports_in_progress)
```

Comment:
nit: can extract `_get_report_count`.

Reply:
I am not sure if that would be a good idea, because `_prepare_display_message` uses `reports_in_progress`. 
If we move it to `_get_report_count` then we might have query redis again.

Reply:
oh i see, can just then
```
@memoized
def _get_active_reports(self):
    report_tracker = CustomSMSReportTracker(self.request.domain)
    return report_tracker.active_reports
```
and then call that in `_prepare_display_message` and use `len(self._get_active_reports())` here.